lib/omnibus/builder.rb

```diff
@@ -644,6 +644,39 @@ def sync(source, destination, options = {})
     expose :sync
 
     #
+    # Helper method to update config_guess in the software's source
+    # directory.
+    # @param [Hash] options
```

Comment:
We should update the comment to mention a `config_guess` dependency is required in any software def that depends on this behavior. See the `bundle` method for an example:
https://github.com/chef/omnibus/blob/94b293b711e3372e04c22f92974d2248a9e60330/lib/omnibus/builder.rb#L317-L318